### PR TITLE
Feature/orgz

### PR DESCRIPTION
**Pull request by jeremyletang**
This is a work in progress, please do not merge.
- [x] create an organization
- [ ] proper handling of mysql db (may need to specify user/passw)
- [ ] dublicate definition of a user (in elastics / mysql)
- [x] create delete join links
- [x] join organization
- [x] get join link from an organization
- [x] get organization
- [x] leave organization
- [x] list organization membership
- [x]  expel from organization

### CODE REVIEW

**Comment by jeremyletang** [comment]
@anas10 @crufter  this is ready to review guys.

**Comment by jeremyletang** [comment]
is it  ok  to merge ? I will need this for  the next steps.

File: daemon/endpoints/organizations.go [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+	// first check if the user is really the organization admin.
+	organizationDao := domain.NewOrganizationDao(db)
+	organization, err := organizationDao.GetById(organizationId)
+
+	if err != nil {
+		return nil, errors.New(fmt.Sprintf("organization %s do not exists", organizationId))
+	}
+	if organization.UserAdminId != userId {
+		return nil, errors.New(fmt.Sprintf(
+			"user (id=%s) is not administrator of organization (id=%s)",
```
Comment (crufter) [inline daemon/endpoints/organizations.go:80]: So an org can have only one admin? I think making it a many to many relation is better for the future

Comment (jeremyletang) [reply daemon/endpoints/organizations.go:80]: yes sure, I was thinking about earlier yesterday.
Maybe we can go with this for now, then I'll update it, so we can start hacking with organization

**Comment by crufter** [comment]
Epic PR! Merged

**Pull request merged**